api/src/main/java/io/opentelemetry/trace/samplers/DefaultSamplingDecision.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace.samplers;
+
+import io.opentelemetry.trace.SamplingDecision;
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Sampling decision without tags.
+ *
+ * @since 0.1.0
+ */
+public class DefaultSamplingDecision implements SamplingDecision {
```

Comment:
Can be moved to `Sampler` interface

Reply:
Can you have an implementation as subclass of an interface?

Reply:
Not sure what you mean. Do you want to move it to `Sampler` interface?